Lesson 5 - CSS/code/exercises/exercise 1/theme.css

```diff
@@ -1,3 +1,86 @@
+body {
+   font-family: 'Times New Roman', Times, serif;
+}
+
 .row {
     display: block;
+    overflow : auto;
+}
+
+.row:last-child h1 {
+    margin-top: 5%;
+    text-align: center;
+}
+
+.row:last-child p {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    margin: 0 5%;
+}
+
+.row:last-child p:before {
+    content: "";
+    background: url("cat.jpg") no-repeat; 
+    display: inline-block;
+    vertical-align:text-bottom;
+    background-size: 60px 60px;
+    padding: 60px 60px 0 0 ;    
+    
+}
+
+.column {
+    display : block;
+    float : left;
+    width : 45%;
+    margin : 0 2%;
+}
+
+
+
+.column * {
+    display : block;
+    margin : 0;
+    padding : 20pt;
+    color : white;
+}
+
+
+.column:first-child h1 {
+    background-color: black;
+    border-radius: 20pt 20pt 0 0;

```

Comment:
This is probably something like `border-radius: 11pt 11pt 0 0;`, but ok in this example it does not matter :)